src/main/java/codeu/model/data/User.java

```diff
@@ -21,7 +21,7 @@
 public class User {
   private final UUID id;
   private final String name;
-  private final String passwordHash;
```

Comment:
This was in order to be able to modify the attributeright?

Reply:
yeah it was, sorry for the late response